### ISSUE TITLE
The removal of the "Adobe Acrobat Reader DC updaters".

### ISSUE THREAD

**Issue by j81blog** [aaronparker/Evergreen#123]
Hmm personally I'm not happy with the removal of the "Adobe Acrobat Reader DC updaters". We have a customized base version and update it with the latest update. And use this for several deployments. Is it possible to somehow get the updates back? I do get your remark but then to just remove it?

_Originally posted by @j81blog in https://github.com/aaronparker/Evergreen/issues/122#issuecomment-791518897_

Perhaps you could use info from the releasenotes? 

```PowerShell
$AdobeVersion = Invoke-WebRequest -UseBasicParsing -Uri "https://armmf.adobe.com/arm-manifests/mac/AcrobatDC/reader/current_version.txt" | Select-Object -ExpandProperty Content
$AdobeInfo = Invoke-WebRequest -UseBasicParsing -Uri "https://www.adobe.com/devnet-docs/acrobatetk/tools/ReleaseNotesDC/index.html" | Select-Object -ExpandProperty Content
$AdobeInfo -match "$AdobeVersion\s(?<Update>[A-Za-z ]*?) update"
$Matches.Update
```

**Comment by DanGough** [comment]
I agree that most using this are primarily interested in the MSP file - but it is easy enough to extract these from the .exe using 7-Zip. Powershell's Expand-Archive can't handle this task though even if you rename it to .zip, but there is a command line version of 7-Zip.

But if there's a way to add the updaters back I'm sure many would welcome it, myself included!

**Comment by j81blog** [comment]
Temporarily I'm using the following code to retrieve the updater (partially borrowed from the previous evergreen module).

```PowerShell
$Version = Invoke-WebRequest -UseBasicParsing -Uri "https://armmf.adobe.com/arm-manifests/mac/AcrobatDC/reader/current_version.txt" | Select-Object -ExpandProperty Content
$Version = $Version -replace '\.',$null
$DownloadUri = "http://ardownload.adobe.com/pub/adobe/reader/win/AcrobatDC/$($Version)/AcroRdrDCUpd$($Version)_MUI.msp"
```

**Comment by aaronparker** [comment]
I'm adding the Reader MSP back to `Get-AdobeAcrobat`. The methods that `Get-AdobeAcrobat` and ``Get-AdobeAcrobatReaderDC` are very different, so this makes sense from a code perspective.

However, Adobe doesn't provide a programmatic way for the updater like they do with the installer, so there's no way to determine whether the updater is optional or required.

**Comment by aaronparker** [comment]
A fix has been added in 4ae0ad263a2a16c04bc031a620e9a9d3282001e2
